backend/models/Assignment.js

```diff
@@ -5,32 +5,36 @@ const assignmentSchema = new mongoose.Schema({
     type: String,
     required: true,
   },
-  tags: {
-    type: String,
-    enum: ["PEERGRADED", "CODING"],
-  },
-  body: {
-    type: String,
-  },
-  pointsPossible: {
+  maxGrade: {
     type: Number,
     required: true,
   },
+  dueDate: {
+    type: Date,
+    required: true,
+  },
+  optional: {
+    type: Boolean,
+    default: false,
+  },
+  locked: {
+    type: Boolean,
+    default: true,
+  },
+  instructions: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "file",
+  },
```

Comment:
Can we call this "files" instead? It seems more inclusive, there might be more than instructions in the files associated with an assignment

Reply:
Makes sense we can do files. However do you wanna instead accept zip files for assignment instructions and submissions - this would preserve the directory structure and reduce storage size on the backend.

Reply:
We can zip files from the frontend and pass references to those (S3 URLS) to the backend as files. You're right that files can absolutely be compressed here